apps/yapms/src/lib/components/modals/toolsmodal/tools/tablecomponents/CandidateActions.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+	import type { Region } from '$lib/types/Region';
+
+	import { get } from 'svelte/store';
+	import { TossupCandidateStore } from '$lib/stores/Candidates';
+	import { CandidatesStore } from '$lib/stores/Candidates';
+	import { RegionsStore } from '$lib/stores/regions/Regions';
+	import { makePatternCSS } from '$lib/utils/patterns';
+	import { splitRegion } from '$lib/stores/regions/regionActions';
+	import Sliders from '$lib/icons/Sliders.svelte';
+
+	export let region: Region;
+
+	$: maxValue = region.candidates.reduce(
+		(prev, current) => (prev > current.count ? prev : current.count),
+		region.candidates[0].count
+	);
+
+	$: winners = region.disabled
+		? [
+				{
+					candidate: $TossupCandidateStore,
+					count: 0,
+					margin: 0
+				}
+			]
+		: region.candidates.filter((candidate) => candidate.count === maxValue);
+
+	function getWinnerId(region: Region) {
+		//if region is split, then select value is "split"
+		if (checkIfSplit(region)) {
+			return 'split';
+		}
+
+		return winners[0].candidate.id;
+	}
+
+	function checkIfSplit(region: Region) {
+		for (const candidate of region.candidates) {
+			if (candidate.count == region.value) {
+				return false;
+			}
+		}
+		return true;
+	}
+
```

Comment:
getWinningColor below just gets how the region is filled on the map.

There is an edge case here if the region is filled with stripes, as you must fill an HTML element with a linear-gradient instead and not an svg <pattern> (and vice versa). Hence, makePatternCSS.